test/unit.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const Promise = require('bluebird');
+const expect = require('chai').use(require('chai-as-promised')).expect;
```

Comment:
chai-as-promised works very well with async functions -- when we get babel running, we'll be able to do something like this:

```javascript
it('does the thing', async () => {
  await doSomething();
  expect(thing).to.equal(otherThing);
});
```

...and there won't be any need to worry about callbacks or done() functions or anything like that.